lib/bundles/base.js

```diff
@@ -70,22 +70,8 @@ require('qtip2');
 require('qtip2/dist/jquery.qtip.css');
 
 // color input
-require('@eastdesire/jscolor');
-jscolor.presets.default = {
-   'alphaChannel': true,
-   'palette': [
-      '#000000', '#7d7d7d', '#870014', '#ec1c23', '#ff7e26',
-      '#fef100', '#22b14b', '#00a1e7', '#3f47cc', '#a349a4',
-      '#ffffff', '#c3c3c3', '#b87957', '#feaec9', '#ffc80d',
-      '#eee3af', '#b5e61d', '#99d9ea', '#7092be', '#c8bfe7',
-   ],
-   'paletteCols': 15,
-   'padding': 8,
-   'borderRadius': getComputedStyle(document.documentElement).getPropertyValue('--input-border-radius').trim(),
-   'backgroundColor': getComputedStyle(document.documentElement).getPropertyValue('--input-bg').trim(),
-   'borderColor': getComputedStyle(document.documentElement).getPropertyValue('--input-border').trim(),
-   'controlBorderColor': getComputedStyle(document.documentElement).getPropertyValue('--input-border').trim(),
-};
+require('spectrum-colorpicker2/dist/spectrum.css');
+require('spectrum-colorpicker2/dist/spectrum');
```

Comment:
`dist/spectrum.js` is targetted by main, should work using this:
```suggestion
require('spectrum-colorpicker2');
```